google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java

```diff
@@ -335,26 +336,27 @@ public void testListMetricsAsync() throws ExecutionException, InterruptedExcepti
 
   @Test
   public void testWriteAndListLogEntries() throws InterruptedException {
-    String logName = formatForTest("test-write-log-entries-log");
-    String filter =
-        "logName = projects/" + logging().getOptions().getProjectId() + "/logs/" + logName;
+    String logId = formatForTest("test-write-log-entries-log");
+    LoggingOptions loggingOptions = logging().getOptions();
+    LogName logName = LogName.create(loggingOptions.getProjectId(), logId);
     StringPayload firstPayload = StringPayload.of("stringPayload");
     LogEntry firstEntry = LogEntry.newBuilder(firstPayload)
         .addLabel("key1", "value1")
-        .setLogName(logName)
+        .setLogName(logId)
```

Comment:
I think we should pass logId, and logName will be constructed (with logId encoded and resourceName prepended):
https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java#L500